homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/mathoperation/MathOperationsDemo.java

```diff
@@ -0,0 +1,32 @@
+package com.tutrit.java.quickstart.homework.mathoperation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MathOperationsDemo {
+
+    Logger log = LoggerFactory.getLogger("MathOperations ");
+
+    int sum(int a, int b) {
```

Comment:
do it as public methods

Reply:
Best practice explicitly specify the access modifier for methods

Reply:
fixed